runtime/support.py

```diff
@@ -160,6 +161,16 @@ def _extract_to_known_location(archive_path, zip_file, extract_dir):
     _log('# extracting %r to %r' % (archive_path, extract_dir))
     zip_file.extractall(extract_dir)
 
+    _log('# byte-compiling extracted unpar directory')
+    quiet = 0 if sys.flags.verbose else 2
+    # By default this will compile at the opt-level used to run the executable
+    # TODO: should we bother using `workers=` parameter here? Probably this
```

Comment:
this is run at RPM `%post` time (which in the future will just be build time in the image-based world,) so it's probably nbd to leave it as the default of `1`

Reply:
Yeah, I suppose so. For the most part we treat unpar extraction as a one-time operation so it's not really that big a deal if it's a little slower.

In systems scrum we actually discussed if it might make sense to extract later for IBU rather than at build time to help shrink down the size of IBU tars, but that is a separate discussion and shouldn't really be impacted by this change AFAIK

Reply:
Where do those `_log` outputs go? Woudl it make sense to have "done byte-compiling" message for easier field-measurement?

Reply:
perhaps. Those messages only get printed when running under `PYTHONVERBOSE=1` so they're really meant as debug logs. Since we also set `quiet=0` a bunch of logs will also be printed as the compilation happens, but it's pretty noisy. I'd probably typically rely on the application doing its own startup stuff to see whenever this gets done